### PR TITLE
[RUN-3713] Add Java 21 information (lts/9.24)

### PR DESCRIPTION
**Pull request by atennapel**
This moves relevant parts of https://github.com/mendix/docs/pull/7585 to refguide9.

Likely Java 21 support will be released with 9.24.22, but this is not certain yet.

### CODE REVIEW

**Review by MarkvanMents — APPROVED** [review]
Thanks for the PR, Albert.
I've proofread it in these commits: https://github.com/mendix/docs/pull/7711/files/f0629a16584609b73ac9d1cf8aa99ca273c5661e..9f8580d7d973a31e9e7acc9ff212e71c2a2f9a65
Approving and will publish with 9.24.22 if it is in that release. Please let me know if it doesn't make that release and I'll update the label.

**Comment by atennapel** [comment]
Thanks Mark! This PR also still needs a review from anoter team member.

**Comment by atennapel** [comment]
@MarkvanMents FYI, this will not make the 9.24.22. Please update this MR to 9.24.23. Thanks.

**Comment by MarkvanMents** [comment]
> @MarkvanMents FYI, this will not make the 9.24.22. Please update this MR to 9.24.23. Thanks.

Updated label. Thanks for the heads-up.

File: content/en/docs/refguide9/installation/jdk-installation.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+* [JDK 11](https://adoptium.net/temurin/releases/?version=11)
+* [JDK 17](https://adoptium.net/temurin/releases/?version=17)
+* [JDK 21](https://adoptium.net/temurin/releases/?version=21)
+
+After downloading execute the downloaded installation executable and follow the instructions.
+
+## 3 Configuring Studio Pro
+
+After the JDK is installed you can configure it in Studio Pro in the [Preferences](/refguide/preferences-dialog/#jdk).
```
Comment (MarkvanMents) [inline content/en/docs/refguide9/installation/jdk-installation.md]: Should still point to refguide9
```suggestion
After the JDK is installed you can configure it in Studio Pro in the [Preferences](/refguide9/preferences-dialog/#jdk).
```

**Review by MarkvanMents — APPROVED** [review]
Update looks good. Just a bad link which I've corrected.

File: content/en/docs/refguide9/modeling/app-explorer/app/app-settings/_index.md [hunk 2]
```diff
@@ -39,7 +39,20 @@ If this option is enabled (**true** by default), Mendix analyzes every microflow
...
+### 3.3 Java Version{#java-version}
+
+Here you can select which Java version to use for you application.
+
+{{% alert color="info" %}}
+For Studio Pro versions 9.24.19 and above, you can choose Java 17.
+For Studio Pro versions 9.24.23 and above, you can choose Java 21.
+{{% /alert %}}
+
+For local development the Java version configured here needs to have a corresponding JDK configured in the [Studio Pro preferences](/refguide/preferences-dialog/#jdk).
```
Comment (MarkvanMents) [inline content/en/docs/refguide9/modeling/app-explorer/app/app-settings/_index.md]:
```suggestion
For local development the Java version configured here needs to have a corresponding JDK configured in the [Studio Pro preferences](/refguide9/preferences-dialog/#jdk).
```

**Review by jveldhuizen — APPROVED** [review]
Looks good - approved on behalf of Runtime Core.

**Pull request merged**